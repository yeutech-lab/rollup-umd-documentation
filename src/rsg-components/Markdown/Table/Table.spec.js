import React from 'react';

import { Table, TableHead, TableBody, TableRow, TableCell } from './index';

describe('Markdown Table', () => {
	it('should render a table', () => {
		const actual = render(
			<Table>
				<TableHead>
					<TableRow>
						<TableCell header>1st header</TableCell>
						<TableCell header>2nd header</TableCell>
					</TableRow>
				</TableHead>
				<TableBody>
					<TableRow>
						<TableCell>1st cell</TableCell>
						<TableCell>2nd cell</TableCell>
					</TableRow>
				</TableBody>
			</Table>
		);

		expect(actual).toMatchSnapshot();
	});
});
