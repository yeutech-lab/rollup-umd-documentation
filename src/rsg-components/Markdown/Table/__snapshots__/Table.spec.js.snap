// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown Table should render a table 1`] = `
<table
  class="rsg--table-0"
>
  <thead
    class="rsg--thead-2"
  >
    <tr>
      <th
        class="rsg--th-4 rsg--td-3"
      >
        1st header
      </th>
      <th
        class="rsg--th-4 rsg--td-3"
      >
        2nd header
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td
        class="rsg--td-3"
      >
        1st cell
      </td>
      <td
        class="rsg--td-3"
      >
        2nd cell
      </td>
    </tr>
  </tbody>
</table>
`;
